apps/token-manager/app/src/components/InfoBoxes.js

```diff
@@ -110,13 +111,17 @@ function InfoBoxes({
           items={stakes}
           renderLegendItem={({ item: account }) => {
             const isCurrentUser = addressesEqual(account, connectedAccount)
+            const [label, showLocalIdentityModal] = useIdentity(account)
```

Comment:
We generally leave the "trailing" arguments if they're not used:

```suggestion
            const [label] = useIdentity(account)
```